regression/cbmc-java/lambda2/test_no_crash_ecj.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+StaticMethodRef.class
+--function StaticMethodRef.test2
+^EXIT=0
+^SIGNAL=0
+--
+--
+Tests that it doesn't crash when failing to read a `BootstrapMethods` entry and
```

Comment:
Is this compiled with eclipse compiler? If so probably worth making a note of that here to stop people inadvertently updating it.

Reply:
It appears to be - will update docs